client/src/app/api/model/instance.ts

```diff
@@ -28,21 +28,31 @@ export const ComponentTypeEnum = {
 
 export interface Instance {
   id?: number;
-  host: string;
-  portNumber: number;
-  name: string;
+  host?: string;
```

Comment:
why did you change the interface? according to the model class used in the [ir](https://github.com/delphi-hub/delphi-registry/blob/develop/src/main/scala/de/upb/cs/swt/delphi/instanceregistry/io/swagger/client/model/Instance.scala) these parameters are not optional in the instance model

Reply:
Initially we didn't change. We just handed over what you deleted without us knowing about it and using it in our code

Reply:
I'm afraid I don't quite understand this answer could you please explain a bit more what you mean?